src/pds_doi_service/core/util/initialize_production_deployment.py

```diff
@@ -65,16 +65,17 @@
 
 from pds_doi_service.core.db.transaction_builder import TransactionBuilder
 from pds_doi_service.core.entities.exceptions import CriticalDOIException
-from pds_doi_service.core.entities.exceptions import InputFormatException
-from pds_doi_service.core.entities.exceptions import UnknownNodeException
+from pds_doi_service.core.entities.exceptions import InputFormatException  # noqa
+from pds_doi_service.core.entities.exceptions import UnknownNodeException  # noqa
 from pds_doi_service.core.outputs.doi_record import CONTENT_TYPE_JSON
 from pds_doi_service.core.outputs.osti.osti_web_parser import DOIOstiXmlWebParser
 from pds_doi_service.core.outputs.service import DOIServiceFactory
 from pds_doi_service.core.outputs.service import SERVICE_TYPE_DATACITE
 from pds_doi_service.core.outputs.service import VALID_SERVICE_TYPES
 from pds_doi_service.core.util.config_parser import DOIConfigUtil
 from pds_doi_service.core.util.general_util import get_logger
-from pds_doi_service.core.util.node_util import NodeUtil
+
+# from pds_doi_service.core.util.node_util import NodeUtil
```

Comment:
Looks like you could just remove this line altogether

Reply:
Right, I did it, thanks